### PR TITLE
Fix staticcheck failures of pkg/probe/http

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix staticcheck failures of pkg/probe/http

**Which issue(s) this PR fixes**:

Ref #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/probe/http/http_test.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ func TestHTTPProbeProxy(t *testing.T) {
 
 	go func() {
 		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-			fmt.Fprintf(w, res)
```
Comment (SataQiu) [inline pkg/probe/http/http_test.go]: pkg/probe/http/http_test.go:76:19: printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)

File: pkg/probe/http/http_test.go [hunk 2]
```diff
@@ -73,7 +73,7 @@ func TestHTTPProbeProxy(t *testing.T) {
...
+			fmt.Fprint(w, res)
```
Comment (ash2k) [inline pkg/probe/http/http_test.go]: This could be just `w.Write([]byte(res))`.

Comment (SataQiu) [reply pkg/probe/http/http_test.go]: Thanks! Your suggestion is good, it looks clearer.

**Review by fejta — APPROVED** [review]
/lgtm
/approve

File: pkg/probe/http/http_test.go (hunk 1 above)
Comment (fejta) [reply pkg/probe/http/http_test.go]: Wouldn't `fmt.Fprint(w, res)` be the closer conversion here?

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Comment by SataQiu** [comment]
> Wouldn't fmt.Fprint(w, res) be the closer conversion here?

Yes @fejta !
I fixed it as you said firstly, but @ash2k suggested a more concise approach:
https://github.com/kubernetes/kubernetes/pull/87646#pullrequestreview-350692142
Maybe it's more efficient to output directly. 
But anyway, it doesn't affect the correctness of the program, right.

**Comment by SataQiu** [comment]
/assign @yujuhong 
Could you please approve this PR? thx~

**Comment by fejta** [comment]
/assign @Random-Liu @dchen1107 @derekwaynecarr @tallclair @vishh @yujuhong

Looking for approval!

File: pkg/probe/http/http_test.go (hunk 1 above)
Comment (tallclair) [reply pkg/probe/http/http_test.go]: yes, please fix.

**Review by fejta — COMMENTED** [review]
/lgtm

**Pull request merged**